tests/Type/data/eloquent-builder.php

```diff
@@ -432,6 +432,11 @@ function testPaginateItems()
     assertType('array<App\User>', User::query()->paginate()->items());
 }
 
+function testChunkOnEloquentBuilder()
+{
+    User::chunk(1000, fn ($collection) => assertType('Illuminate\Database\Eloquent\Collection<App\User>'));
```

Comment:
After the above change, this needs to be updated also for the 2 generic types.

Reply:
And done!